app/src/main/java/com/framgia/englishconversation/screen/timeline/TimelineAdapter.java

```diff
@@ -1,23 +1,36 @@
 package com.framgia.englishconversation.screen.timeline;
 
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+
 import com.framgia.englishconversation.data.model.MediaModel;
 import com.framgia.englishconversation.data.model.TimelineModel;
 import com.framgia.englishconversation.databinding.ItemTimelineAudioBinding;
 import com.framgia.englishconversation.databinding.ItemTimelineConversationBinding;
 import com.framgia.englishconversation.databinding.ItemTimelineImageBinding;
 import com.framgia.englishconversation.databinding.ItemTimelineOnlyTextBinding;
 import com.framgia.englishconversation.databinding.ItemTimelineVideoBinding;
+import com.google.android.exoplayer2.ui.SimpleExoPlayerView;
+
+import java.net.URL;
 import java.util.List;
 
+import im.ene.toro.ToroPlayer;
```

Comment:
ERROR: Unused import: im.ene.toro.ToroPlayer. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports